include/behaviortree_cpp/bt_factory.h

```diff
@@ -44,13 +46,11 @@ template <typename T>
 inline TreeNodeManifest CreateManifest(const std::string& ID,
                                        PortsList portlist = getProvidedPorts<T>())
 {
-  if constexpr( has_static_method_description<T>::value)
+  if constexpr( has_static_method_metadata<T>::value )
   {
-    return {getType<T>(), ID, portlist, T::description()};
-  }
-  else {
-    return {getType<T>(), ID, portlist, {}};
+    return {getType<T>(), ID, portlist, T::metadata()};
   }
+  return {getType<T>(), ID, portlist, {}};
```

Comment:
not that it matters, but the previous version of the code explicitly used if-else instead of relying on the `if` condition to return. Of course, it's up to what the maintainer prefers :)